Gemfile

```diff
@@ -2,6 +2,8 @@ source 'https://rubygems.org'
 ruby "2.1.5"
 git_source(:github) { |repo_name| "https://github.com/#{repo_name}.git" }
 
+gem 'ofn_web', path: 'apps/web'
```

Comment:
I would call it engines to keep the convention. I guess it'll make it easier for new Rails devs to join.

Reply:
done 👍